### PR TITLE
Update versioning to 2.5.2

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview

Update versioning to 2.5.2

### CODE REVIEW

**Review by davidkline-ms — APPROVED** [review]
looks good!

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -7,9 +7,8 @@
 
 > [!IMPORTANT]
 > There is a known compiler issue that impacts applications built for Microsoft HoloLens 2 using
-> ARM64. This issue is fixed by updating Visual Studio 2019 to version 16.8 or later. If you are unable to update Visual Studio, 
-> please import the `com.microsoft.mixedreality.toolkit.tools` package to apply 
-> a workaround.
+> ARM64. This issue is fixed by updating Visual Studio 2019 to version 16.8 or later. If you are unable to update Visual Studio,
+> please import the `com.microsoft.mixedreality.toolkit.tools` package to apply a workaround.
 
 ## What's new
```
Comment (MaxWang-MS) [inline Documentation/ReleaseNotes.md:14]: Do we want to update this part as well?

Comment (keveleigh) [reply Documentation/ReleaseNotes.md:14]: Good catch! I'm updating the rest of the sections in https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9060, alongside the changes

**Pull request merged**